static/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -331,9 +331,18 @@ tags.sku_class:enterprise #enterprise`;
                   fields: [
                     {
                       name: 'autoAssignment',
-                      type: 'boolean',
-                      label: t('Automatically assign issues'),
-                      help: t('Assign issues when a new event matches the rules above.'),
+                      type: 'choice',
+                      label: t('Prioritize Auto Assignment'),
+                      help: t(
+                        "When there's a conflict between suspect commit and ownership rules."
+                      ),
+                      choices: [
+                        [
+                          'Auto Assign to Suspect Commits',
+                          t('Auto assign to suspect commits'),
+                        ],
+                        ['Assign To Issue Owner', t('Assign to issue owner')],
```

Comment:
Should this be "Auto Assign to Issue Owner" based on the text in `static/app/types/group.tsx`?